hiss/application/tests/view_tests/create.py

```diff
@@ -41,6 +41,8 @@ def test_associates_with_user(self) -> None:
         self.client.force_login(self.user)
         self.create_active_wave()
 
+        self.application_fields["school"] = self.first_school.pk
```

Comment:
This `self.application_fields["school"]` snippet occurs a bunch of times. We should figure out a better way to write this so we don't have to do it over and over again.

Reply:
what exactly is self.first_school.pk?

Reply:
[It stands for primary key](https://docs.djangoproject.com/en/3.0/topics/db/queries/#the-pk-lookup-shortcut)

Reply:
I will try to think of ways of reducing the amount of times to write this